src/app/pages/index/components/list-requests/list-requests.component.ts

```diff
@@ -1,16 +1,66 @@
 import { Component, OnInit } from '@angular/core';
+import { DebugService } from '../../../../services/debug.service';
+import { MatIconRegistry } from '@angular/material/icon';
+import { DomSanitizer } from '@angular/platform-browser';
+import { Debug } from '../../../../models/Debug';
 
 @Component({
   selector: 'app-list-requests',
   templateUrl: './list-requests.component.html',
   styleUrls: ['./list-requests.component.css'],
 })
 export class ListRequestsComponent implements OnInit {
+  loading: boolean = false;
+  debugsList: Debug[] = [];
+  displayedColumns: string[] = ['position', 'tag', 'ip', 'method', 'isAjax', 'url', 'code', 'memory', 'time'];
 
-  constructor() {
+  constructor(private debugService: DebugService,
+              iconRegistry: MatIconRegistry,
+              sanitizer: DomSanitizer) {
+    iconRegistry.addSvgIcon(
+      'check',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/img/yes-icon.svg'));
+    iconRegistry.addSvgIcon(
+      'ban',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/img/no-icon.svg'));
   }
 
   ngOnInit(): void {
+    this.getDebugsList();
   }
 
+  getDebugsList() {
+    this.loading = true;
+    this.debugService.getList().subscribe(
+      response => {
+        this.loading = false;
+        this.debugsList = response;
+      }
+    );
+  }
+
+  formatMemory(memory: number) {
+    const val: number = Math.round(memory / 10485.76) / 100;
```

Comment:
```suggestion
    const val: number = (new Number(memory / 10485.76)).toPrecision(4) / 100;
```

Reply:
I have slightly changed your approach, but anyway thanks for the good suggestion!

Reply:
Although there must be at least 2 decimal places.